misc/geolite-update.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# set -xe
+
+# geolite-update.sh -- update GeoIP lite databases.
+# (C) 2017 Ettercap Development Team.
+
+# Ettercap can use MaxMind's GeoIP databases to look up the country for an IP address.
+# This simple shell script helps in downloading and installing the *free* GeoLite
+# country databases, which are needed for this feature to actually work.
+# You can optionally pass an alternative install path to this script.
+# If you rather not, it will install all files to /usr/local/share/GeoIP/.
```

Comment:
Can you adjust the default path of the *geoip_data_file* option in the **etter.conf** file.

Reply:
Sure.